deploy/terraform-datacommons-website/modules/apikeys/main.tf

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# The following resource currently has issues with ADC authentication.
```

Comment:
This and some other caveat make me think if we should reconsider, at least part of the approach here. Eventually maybe Terraform can only handle part of the setup, either initial or deployment. We may anyway need some manual setup? It's an open question.

Reply:
That's a good point. Theoretically there are always workarounds and even the workarounds imo are easier to manage than raw scripts, the upgrade story can get messy. 

I think we'll have a better idea of how this works when there's an actual need to set up new instances whether by us or by other users.